src/main/java/com/github/gilday/darkmode/DarkModeSync.java

```diff
@@ -23,16 +24,18 @@
  */
 public final class DarkModeSync implements Disposable {
 
+  private static final Logger logger = Logger.getInstance(DarkModeSync.class);
   private final ScheduledFuture<?> scheduledFuture;
-
   private final LafManager lafManager;
   private final DarkModeSyncThemes themes;
+  private boolean isDarkMode;
```

Comment:
I believe this is only used in the `updateLafIfNecessary` method, so let's make it a local variable there

Reply:
Changed.